guides-markdown/regtest.md

```diff
@@ -0,0 +1,76 @@
+# Create a Local Regtest Blockchain
+
+```post-author
+Jonathan Gonzalez
+```
+```post-description
+This example creates a local Regtest Chain.
+```
+
+## Regtest Chain Example
+The Regtest Network is a local Blockchain in which individuals can almost
+instantaneously generate blocks on demand. Including the option to  generate private satoshis
+with no real-world value. This is ideal for testing Bitcoin Applications.
+
+This example generates two consecutive blocks, with transaction confirmations
+in the Users WalletDB:
+
+
+```javascript
+const FullNode = require('bcoin/lib/node/fullnode');
```

Comment:
lets make the imports a little more generalized for different setups. In most of the guides we usually forst do `const bcoin = require('bcoin')` and then pull the modules from there (e.g. `const FullNode = bcoin.fullnode;` etc.) that way, people just need to change one line depending on where their version of bcoin is installed.